mula/scheduler/schedulers/scheduler.py

```diff
@@ -440,6 +416,30 @@ def stop_threads(self) -> None:
 
         self.threads = []
 
+    def is_enabled(self) -> bool:
```

Comment:
I'm not sure if this is just style, or something else, but this method seems unneeded. If it where more complex I'd still expect it to be a property, but since it just returns a property already that doesn't really add anything.

Reply:
I'll pick this up in a different pr, since this one was mainly about the documentation. In this file I mainly changed the location of the methods around.